client/src/themes/theme.js

```diff
@@ -2,9 +2,9 @@ import { createMuiTheme } from "@material-ui/core";
 
 export const theme = createMuiTheme({
   typography: {
-    fontFamily: '"Roboto"'
+    fontFamily: '"Heiti SC, Light"'
```

Comment:
You'll need to add this font to your index.html file - the fonts are not rendering properly

Reply:
@skassam21 I couldn't find Heiti SC, Light on google api - only Adobe. Is there a better alternative?